randovania/gui/widgets/multiplayer_session_users_widget.py

```diff
@@ -161,6 +161,12 @@ async def _prompt_for_preset(self, default_game: RandovaniaGame | None):
     @asyncSlot()
     async def _world_replace_preset(self, world_uid: uuid.UUID):
         game = self._session.get_world(world_uid).preset.game
+
+        for user in self._session.users.values():
+            if user.worlds.get(world_uid, None) is not None:
```

Comment:
Default value of get is None. Also I think this could've been just an `in` check